src/common/color-element.js

```diff
@@ -39,7 +39,16 @@ const Self = class ColorElement extends NudeElement {
 		// Hide elements before they are defined
 		let style = document.getElementById("color-element-styles")
 		          ?? Object.assign(document.createElement("style"), {id: "color-element-styles"});
-		style.textContent = `:is(${ colorTags.join(", ") }):not(:defined) {display: none}`;
+		style.textContent = `
+			:is(${ colorTags.join(", ") }) {
+				opacity: 0;
+
+				body:not(.loading-global-color-element-styles) &:defined {
```

Comment:
Also, ideally this should be solved in Nude Element (via a mixin): components should define their dependencies via a static property, and then these styles are generated for all dependencies.